duckduckgo/iOS
# Update to SwiftLint 0.50.1

Task/Issue URL: https://app.asana.com/0/0/1203440314082171/f
Tech Design URL:
CC:

**Description**:

This PR updates the codebase to use SwiftLint 0.50.1.



**Steps to test this PR**:
1. Run this branch with SwiftLint 0.50.0 and validate that it doesn't produce any warnings
2. Check that there are no warnings from Xcode 14 about build scripts



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
